tests/parametric/test_parametric_endpoints.py

```diff
@@ -0,0 +1,732 @@
+# TODO(mabdinur): Move this file to tests/test_the_test/
+"""
+This module provides one unit test for each parametric endpoint.
+The results of these unit tests are reported to the feature parity dashboard.
+Parametric endpoints that are not tested in this file are not yet supported.
+Avoid using these endpoints in the parametric tests.
```

Comment:
```suggestion
Avoid using those endpoints in the parametric tests.
```
Should we list those? Or should we just delete this line knowing that we intend on adding tests for them. I feel like if I found an "unsupported" endpoint that I needed... I'd probably just use it anyways... or maybe would write the test for it.

Reply:
Nah we shouldn't list, it's meant to be a simple warning. I plan to link this file in our onboarding docs. We should clearly state that all new endpoints should have a unit test here.